icepyx/core/variables.py

```diff
@@ -482,25 +492,56 @@ def append(self, defaults=False, var_list=None, beam_list=None, keyword_list=Non
         To add all variables and paths in ancillary_data
 
         >>> reg_a.order_vars.append(keyword_list=['ancillary_data']) # doctest: +SKIP
+
+        To add variables as a list of explicit paths
+
+        >>> path_list = ['gt1l/heights/delta_time',
+        ...              'gt1l/heights/dist_ph_across',
+        ...              'gt1l/heights/dist_ph_along',
+        ...              'gt1l/heights/h_ph',
+        ...              'gt1l/heights/lat_ph',
+        ...              'gt1l/heights/lon_ph']
+        >>> region_a.order_vars.append(path_list=path_list)  # doctest: +SKIP
+
         """
 
-        assert not (
-            defaults == False
-            and var_list == None
-            and beam_list == None
-            and keyword_list == None
-        ), "You must enter parameters to add to a variable subset list. If you do not want to subset by variable, ensure your is2.subsetparams dictionary does not contain the key 'Coverage'."
+        # Check that at least one allowable combination of keywords are set
+        if (
+            not defaults
+            and not (var_list or beam_list or keyword_list)
+            and not path_list
+        ):
+            raise ValueError(
+                "Either default or path_list, or at least one of var_list, "
+                "beam_list or keyword_list must be set."
+            )
+
+        # Check that only path_list, or only var_list, beam_list or keyword_list
+        # are set.
+        if (var_list or beam_list or keyword_list) and path_list:
+            raise ValueError(
+                "path_list cannot be set if var_list or "
+                "beam_list or keyword_list are set"
+            )
 
         final_vars = {}
 
+        if path_list:
+            _, (beams, keywords, variables) = self.parse_var_list(
+                path_list, tiered_vars=True
+            )
+            var_list = list(set(variables))
+            beam_list = list(set(beams))
+            keyword_list = list(set(keywords))
```

Comment:
I went to add the new example in the docstring to the variables tutorial notebook it raised 2 issues:
1. The example fails because it's for ATL03, and the examples are on ATL06 (and in the notebook, there's a second one for ATL09)
2. Trying to use a `path_list` input on ATL06 fails because it's got four layers of variables, not the three we collect here.

I haven't reminded myself of how we handle this in general in the module, so that may recommend a solution for (2). (1) is easy if we just sub in another list.